index.js

```diff
@@ -277,6 +277,21 @@
     return Nothing();
   });
 
+  //. pluck :: String -> [Object] -> [Maybe *]
+  //.
+  //. Takes a list of objects and plucks the value of the specified key
+  //. for each object in the list. Returns the value wrapped in a Just
+  //. if an object has the key and a Nothing if it does not.
+  //.
+  //. ```javascript
+  //. > pluck('a', [{a: 1, b: 2}, {a: 4, b: 5}, {b: 3, c: 7}])
+  //. [Just(1), Just(4), Nothing()]
+  //. ```
+
+  S.pluck = R.curry(function (key, xs) {
+    return R.map(S.get(key), xs);
+  });
```

Comment:
> any though about using `prop` instead for tighter compatibility?

I've thought about it, certainly. I prefer "get" though as it's a verb. It's nice to be able to talk about

```javascript
R.pipe(S.get('a'),
       R.chain(S.get('b')),
       R.chain(S.get('c')))
```

as "chaining gets". #13 added `S.gets` which does multiple "gets". As a noun, "prop" does not work as well for these cases.